docs/deployment/deployHelmLocalIngress.md

```diff
@@ -0,0 +1,103 @@
+
+```sh
+127.0.0.1 myping-pingaccess-admin.ping-local.com ...
+```
+
+When using our charts, the release name provided to helm is prepended - that is what provides the `myping-` portion of the hostname in the file.  The `ping-local.com` domain suffix is provided through the ingress definitions as shown later on this page.  So, the structure is:
+
+```sh
+<helm-release-name>-<ping-product-service>.<domain-name-from-ingress>
+```
+
+If you use a release name other than `myping` or a domain other than `ping-local.com` you will need to update the aliases in `/etc/hosts`/ accordingly.
+
+## Instructions
+
+There is a file under the `30-helm` directory of this repository named `ingress.yaml`.  Modify this file for use with a local cluster:
+
+* Replace `insert domain name here` with your domain name (ping-local.com in this guide)
+* Edit line 11, removing the `-public` suffix for the class
+
+The file should look as shown here:
+
+```yaml
+global:
+  envs:
+    PING_IDENTITY_ACCEPT_EULA: "YES"
+  ingress:
+    enabled: true
+    addReleaseNameToHost: prepend
+    defaultDomain: "ping-local.com"
+    defaultTlsSecret:
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+      kubernetes.io/ingress.class: "nginx"
+```
+
+## Create ingresses
+
+When deploying using helm and one of the example YAML files in the 30-helm directory, also pass in the ingress.yaml file to include the ingress definitions as part of the overall deployment.
+
+For example, to use the `everything.yaml` and include ingresses, you would run the following command from the 30-helm directory (after updating the ingress.yaml file):
+
+```sh
+helm upgrade --install myping pingidentity/ping-devops -f everything.yaml -f ingress.yaml
+```
+
+After everything has started, you will see the same pods and services as in the getting started example.  In addition, you will see ingress definitions in the same namespace as the products:
+
+```sh
+# List ingress definitions
+kubectl get ingress
+
+# Output
+NAME                         CLASS    HOSTS                                       ADDRESS     PORTS     AGE
+myping-pingaccess-admin      <none>   myping-pingaccess-admin.ping-local.com      localhost   80, 443   47m
+myping-pingaccess-engine     <none>   myping-pingaccess-engine.ping-local.com     localhost   80, 443   47m
+myping-pingauthorize         <none>   myping-pingauthorize.ping-local.com         localhost   80, 443   47m
+myping-pingdataconsole       <none>   myping-pingdataconsole.ping-local.com       localhost   80, 443   47m
+myping-pingdirectory         <none>   myping-pingdirectory.ping-local.com         localhost   80, 443   47m
+myping-pingfederate-admin    <none>   myping-pingfederate-admin.ping-local.com    localhost   80, 443   47m
+myping-pingfederate-engine   <none>   myping-pingfederate-engine.ping-local.com   localhost   80, 443   47m
+```
+
+The HOSTS column reflects the entries added to the `/etc/hosts' file.
```

Comment:
Mismatched backtick and single quote here

Reply:
fixed